MathGame.destocot/MathGame/Helpers.cs

```diff
@@ -0,0 +1,127 @@
+﻿using MathGame.Models;
+
+namespace MathGame
+{
+    internal class Helpers
+    {
+        internal static List<Game> games = new List<Game>();
```

Comment:
This initialisation can be simplified to:

```suggestion
        internal static List<Game> games = new();
```